Sources/AppDebugMode/Views/AppDebugView.swift

```diff
@@ -89,6 +89,11 @@ struct AppDebugView<CustomControls: View>: View {
                 title: "Connections",
                 image: Image(systemName: "network"),
                 destination: AnyView(erasing: ConnectionsSettingsView())
+            ),
+            Screen(
+                title: "Memory",
```

Comment:
nechceme to premenovat nech to sedi s nazvom tej libky?

```suggestion
                title: "Screen lifetime tracker",
```

Reply:
Trocha som dúfal že by som tam pridal aj nejaky memory graph z inej libby ale zatiaľ som nenašiel takže môže byť.